index.js

```diff
@@ -193,6 +193,9 @@ const AvatarEditor = React.createClass({
   },
 
   isDataURL (str) {
+    if (str === null) {
```

Comment:
I think it's ok in this case since the function is named isDataURL, and null is certainly not